boards/nucleo-l152/include/periph_conf.h

```diff
@@ -34,9 +34,17 @@ extern "C" {
  **/
 #define CLOCK_HSI           (16000000U)             /* frequency of internal oscillator */
 #define CLOCK_CORECLOCK     (32000000U)             /* targeted core clock frequency */
-/* 0: no external low speed crystal available,
- * 1: external crystal available (always 32.768kHz) */
-#define CLOCK_LSE           (1)
+/*
+ * 0: no external low speed crystal available,
+ * 1: external crystal available (always 32.768kHz)
+ *
+ * LSE might not be available by default in early (C-01) Nucleo boards.
+ * For newer revisions, LSE crystal is present, but currently is not working.
+ * (issue #8545).
```

Comment:
@kYc0o, use the full url, e.g. https://github.com/RIOT-OS/RIOT/pull/8545

Reply:
It's evident that the comment it's in the context of a github issue. Though I've put the complete address and amended directly the change.